integrations/opennms-vmware/src/test/java/org/opennms/netmgt/collectd/vmware/VmwareViJavaAccessTest.java

```diff
@@ -120,7 +162,12 @@ protected void relax() {
         mockServiceInstance = createMock(ServiceInstance.class);
 
         // setup ServerConnection
-        mockServerConnection = new ServerConnection(new URL("https://hostname/sdk"), new VimPortType(new WSClient("https://hostname/sdk")), mockServiceInstance);
+        mockServerConnection = new ServerConnection(new URL("https://hostname/sdk"), new VimPortType(new WSClient("https://hostname/sdk") {
+            @Override
+            protected SSLSocketFactory getTrustAllSocketFactory(boolean ignoreCert) throws RemoteException {
```

Comment:
For the test setup it does not really matter, as the connection is mocked anway.